objects/Condor.py

```diff
@@ -0,0 +1,55 @@
+import os
+import fileinput
+import asyncio
+from .OPTSubprocess import OPTSubprocess
+
+class Condor(OPTSubprocess):
+    def __init__(self, id, logger, cwd, ip, port, poll_interval, workers, timeout):
+        super().__init__('Condor', id, logger)
+        self.ip = ip
+        self.port = port
+        self.poll_interval = poll_interval
+        self.workers = workers
+        self.timeout = timeout
+        self.cwd = cwd
+    
+    async def init_condor(self):
+        self.modify_timeout()
```

Comment:
I can't really speak to the `async` stuff since I haven't used it before, but looks like a good approach!